debugger-support/briefcase-pdb-debugger-support/src/briefcase_pdb_debugger_support/_remote_debugger.py

```diff
@@ -0,0 +1,30 @@
+import json
+import sys
+
+from remote_pdb import RemotePdb
+
+
+def start_pdb(config_str: str, verbose: bool):
+    """Start remote PDB server."""
+    debugger_config: dict = json.loads(config_str)
+
+    # Parsing host/port
+    host = debugger_config["host"]
+    port = debugger_config["port"]
+
+    print(
+        f"""
+Remote PDB server opened at {host}:{port}, waiting for connection...
+To connect to remote PDB use eg.:
+    - telnet {host} {port} (Windows, Linux)
+    - rlwrap socat - tcp:{host}:{port} (Linux, macOS)
```

Comment:
There's some additional instructions needed here, because neither `rlwrap` and `socat` are commands on my macOS machine. It looks like rlwrap might be a python module ... but the only socat references I can find are for linux commands.